apps/ex_wire/config/dev.exs

```diff
@@ -2,7 +2,10 @@ use Mix.Config
 
 config :ex_wire,
   network_adapter: {ExWire.Adapter.UDP, NetworkClient},
-  sync: false,
+  sync: true,
+  private_key:
+    <<10, 122, 189, 137, 166, 190, 127, 238, 229, 16, 211, 182, 104, 78, 138, 37, 146, 116, 90,
```

Comment:
I wasn't sure if I should leave this hardcoded key here. 

It is often a source of confusion _not_ to have it, because without this we will generate a random key every time we interact with a message from a peer. That means we'll use one key to encrypt secrets and one key to decrypt them. And it is a difficult bug to trace.